### ISSUE TITLE
Publish on Alfred Gallery

### ISSUE THREAD

**Issue by pulsone21** [alexchantastic/alfred-open-with-vscode-workflow#14]
Would be nice to find that workflow on Alfred Gallery ;)

**Comment by alexchantastic** [comment]
Good idea! I've submitted it here: https://www.alfredforum.com/topic/20294-open-with-vscode/. I'll close this once it's officially listed!

**Comment by notDavid** [comment]
While you're at it, please consider adding a Hotkey action for "selection in macOS" to instantly open a file :-)

**Comment by alexchantastic** [comment]
> While you're at it, please consider adding a Hotkey action for "selection in macOS" to instantly open a file :-)

Thanks for the suggestion! I've pulled that into #15.
